src/Symfony/Component/Notifier/Notifier.php

```diff
@@ -71,10 +72,10 @@ private function getChannels(Notification $notification, RecipientInterface $rec
             $errorPrefix = sprintf('Unable to determine which channels to use to send the "%s" notification', \get_class($notification));
             $error = 'you should either pass channels in the constructor, override its "getChannels()" method';
             if (null === $this->policy) {
-                throw new LogicException(sprintf('%s; %s, or configure a "%s".', $errorPrefix, $error, ChannelPolicy::class));
+                throw new LogicException(sprintf('"%s"; "%s", or configure a "%s".', $errorPrefix, $error, ChannelPolicy::class));
```

Comment:
Fixed to pass the CI, I can revert.

Reply:
Yeah. Let's not do unrelated changes. Even though if Fabbot suggest it.

Reply:
@Nyholm should I do a specific PR for this change?

Reply:
this is a wrong error reported by fab it.

To get ride of this, you should switch from spring to string concatenation.